builtin/logical/pki/path_config_acme.go

```diff
@@ -197,6 +205,14 @@ func (b *backend) pathAcmeWrite(ctx context.Context, req *logical.Request, d *fr
 		}
 	}
 
+	if eabPolicyRaw, ok := d.GetOk("eab_policy"); ok {
+		eabPolicy, err := getEabPolicyByString(eabPolicyRaw.(string))
+		if err != nil {
+			return nil, fmt.Errorf("invalid eab policy name provided")
```

Comment:
Maybe list supported values here as well to help the user?

Reply:
Yup that is a good addition, added in https://github.com/hashicorp/vault/pull/20523/commits/5b453138436ebaf79039f8a0c8f3723b56782339